.tours/5---create unlocked packages.tour

```diff
@@ -0,0 +1,25 @@
+{
+    "title": "5 - Create Unlocked Packages",
+    "steps": [
+        {
+            "file": ".github/workflows/packaging-sfdx-ci-develop.yml",
+            "description": "Creation of package versions for each package is automated using GitHub actions.",
```

Comment:
I think we should show how package creation is done. So instead of saying there is automation, we can talk about each command in one step, and say we have used this file to automate this sequence of steps for every deployment.

Reply:
If we revamp the first code tour to show the folder structure, and how files are split in those folders, we can use this tour to talk about sfdx-project.json first.. Show how you define packages for different folders and how to set dependencies between them, and then switch to the GitHub action file and tell how packaging is done.

Reply:
This is a great suggestion! I did change it accordingly, please review.